modules/farm_fd2/src/entrypoints/cover_crop_seeding/lib.submitError.unit.cy.js

```diff
@@ -0,0 +1,222 @@
+import { lib } from './lib.js';
+
+describe('Error when submitting using the cover_crop lib.', () => {
+  /*
+   * Create a form object that has the same format as the data.form
+   * object used in the cover_crop entry point. This will be passed
+   * to the lib functions as if it is coming from the cover crop
+   * entry point as a submission.
+   */
+  let form = {
+    date: '1950-01-02',
+    crops: ['BEAN', 'CARROT'],
+    location: 'ALF',
+    beds: ['ALF-1', 'ALF-3'],
+    areaSeeded: 50,
+    seedApplicationEquipment: ['Tractor'],
+    seedIncorporationEquipment: ['Rake'],
+    seedApplicationDepth: 6,
+    seedApplicationSpeed: 5,
+    seedIncorporationDepth: 8,
+    seedIncorporationSpeed: 3,
+    winterKill: true,
+    winterKillDate: '1950-12-31',
+    comment: 'A comment',
+  };
+
+  beforeEach(() => {
+    cy.restoreLocalStorage();
+    cy.restoreSessionStorage();
+  });
+
+  afterEach(() => {
+    cy.saveLocalStorage();
+    cy.saveSessionStorage();
+  });
+
```

Comment:
I'm a little confused about what the next two tests are checking. I'd like to discuss them when you get to the point of making changes to this PR.